responsibleai/responsibleai/managers/error_analysis_manager.py

```diff
@@ -188,8 +225,7 @@ class ErrorAnalysisManager(BaseManager):
     def __init__(self, model: Any, dataset: pd.DataFrame, target_column: str,
                  classes: Optional[List] = None,
                  categorical_features: Optional[List[str]] = None,
-                 dropped_features: Optional[List[str]] = None,
-                 task_type: Optional[TaskType] = None):
+                 dropped_features: Optional[List[str]] = None):
```

Comment:
passing task_type might still be fine, but default would need to be ModelTask.UNKNOWN instead of None.  If removing it here works that's fine too though, since this is how it was before anyway it seems.

Reply:
yeah we don't pass task_type here before. So removing it as it is not needed